README.md

```diff
@@ -24,18 +24,20 @@ If helpful, [here's a short video](https://github.com/anti-work/shortest/issues/
 
 ### Installation
 
-```bash
```

Comment:
Manual instructions can be removed. The description below outlines the steps, in case someone would like to do it manually.